lib/buckets/common.js

```diff
@@ -11,11 +11,16 @@
 var EventEmitter = require('events').EventEmitter;
 
 var assert = require('assert-plus');
-var jsprim = require('jsprim');
-var sprintf = require('util').format;
+var util = require('util');
+var vasync = require('vasync');
+
+var format = util.format;
 
 require('../errors');
```

Comment:
Sorry, just noticed this funky import as well.

Reply:
This is used to get `InvalidLimitError` defined in this file.  I'll ask about this in ~manta, but I think it would be best to revamp `errors.js` to:

1. not abuse `global`, and
2. not `fs.readFileSync` itself to create exports

as part of a separate PR

Reply:
+1 to that idea, @bahamas10